src/renderer/components/TimelineSpace/Modals/NewToot/Status.vue

```diff
@@ -319,7 +319,6 @@ export default {
     resize: none;
     height: 120px;
     transition: border-color 0.2s cubic-bezier(0.645, 0.045, 9.355, 1);
-    font-family: 'Lato', sans-serif;
```

Comment:
Hmm... The reason of Lato is a bug of Noto Sans.
Noto Sans could not render backticks properly, so I have set Lato. 
https://github.com/h3poteto/whalebird-desktop/issues/382

Reply:
Found the issue, but I can't reproduce it. My Noto Sans version is 2.0 (courtesy of TeX Live). Perhaps it's been updated?

Reply:
I still reproduce it on my computer.
![screenshot from 2018-11-21 21-40-43](https://user-images.githubusercontent.com/4631959/48841945-49199e00-edd6-11e8-8a7d-2419f3edf56f.png)

Reply:
I can reproduce it too *only with the embedded version in Whalebird*.
<img src="https://user-images.githubusercontent.com/13498015/48843536-15a72700-ed76-11e8-8d34-cec2a18b7d4e.png" width="400px"></img>

Noto Sans 2.x works correctly.
<img src="https://user-images.githubusercontent.com/13498015/48843551-23f54300-ed76-11e8-9c92-c8cd797da085.png" width="400px"></img>

Reply:
Oh, I see, thank you.
I'll update Noto Sans in Whalebird.